scripts/fastx_subsample.sh

```diff
@@ -897,7 +900,7 @@ OUTPUT=$(mktemp)
 printf "@s1\nA\n+\nG\n@s2\nA\n+\nG\n" | \
     "${VSEARCH}" --fastx_subsample - --fastqout "${OUTPUT}" --sample_size 2 \
 	&> /dev/null
-    [[ $(wc -l < "${OUTPUT}") == "8" ]] && \
+    [[ $(echo $(wc -l < "${OUTPUT}")) == "8" ]] && \
```

Comment:
I can rewrite the test in a more concise way to remove the need for a storage variable and a tmp file:
```sh
printf "@s1\nA\n+\nG\n@s2\nA\n+\nG\n" | \
    vsearch \
        --fastx_subsample - \
        --sample_size 2 \
        --fastqout - 2> /dev/null | \
    awk 'END {exit NR == 8 ? 0 : 1}' && \
    echo "success" || \
        echo "failure"  ## vsearch's output should be 8 lines of fastq
```
Does it work for you on MacOS @torognes ?

Reply:
I have tested this and these changes work well.